django_app/tests/test_migrations.py

```diff
@@ -77,3 +77,54 @@ def test_0020_remove_chatmessage_source_files_textchunk_and_more(migrator):
 
     # Cleanup:
     migrator.reset()
+
+
+@pytest.mark.django_db()
+def test_0027_alter_file_status(migrator):
+    # Not using test parametrisation to avoid repeatedly rerunning migration
```

Comment:
Happy to change this to test parametrisation if more readable

Reply:
nah not needed. migration tests are long running enough